Configure

```diff
@@ -249,7 +249,7 @@ $config{shlib_version_history} = "unknown";
 
 collect_information(
     collect_from_file(catfile($srcdir,'include/openssl/opensslv.h')),
-    qr/OPENSSL.VERSION.TEXT.*OpenSSL (\S+) / => sub { $config{version} = $1; },
+    qr/^# define OPENSSL.VERSION.NUMBER.TEXT *"([^"]+)"/ => sub { $config{version} = $1; },
```

Comment:
If we're adding all that context, let's make sure that some future reformatting doesn't have this go south:

``` perl
    qr/^#\s+define\s+OPENSSL.VERSION.NUMBER.TEXT\s+"([^"]+)"/ => sub { $config{version} = $1; },
```

Reply:
I've updated this now.